### ISSUE TITLE
Prompts panel: Delete button missing accessible name

### ISSUE THREAD

**Issue by kangabell** [danny-avila/LibreChat#5566]
There is a red button with a trash can icon that triggers a modal to delete a saved prompt. VoiceOver reads this button as "dialog pop up collapsed, button", with no indication to what the dialog is for. Once in the dialog, the focus is moved to a button that reads "Cancel". It's still not clear what the dialog is for until you navigate backwards into the content.	

## Test Environment
http://159.89.83.1/d/prompts

## WCAG Criteria
2.5.3 Label in Name - A	

## Testing Method
Manual - VoiceOver, Chrome, MacOS	

## Screen Recording
https://share.zight.com/ApuoXwyn
Navigating the prompt deletion dialog with a screen reader and keyboard 

## Code Snippet
```<button class="inline-flex items-center justify-center gap-2 whitespace-nowrap text-sm font-medium ring-offset-background focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 text-primary-foreground rounded-lg h-10 w-10 border border-transparent bg-red-600 transition-all hover:bg-red-700 dark:bg-red-600 dark:hover:bg-red-800 p-0.5" type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-:rbt:" data-state="closed"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-trash2 cursor-pointer text-white size-5">...</svg></button>```

## Recommended Action
Add button text or an aria-label that says "delete prompt" or similar. Also, do not move focus to the cancel button once inside the dialog, so that its contents can be read first.	

## Notes
After deleting from this dialog, the dialog closes and the screen reader says the name of the prompt that was just deleted. It should probably instead say "[prompt name] deleted" to clarify that the action worked.
